integration-tests/smoke/auto_ocr_test.go

```diff
@@ -0,0 +1,181 @@
+package smoke
+
+import (
+	"context"
+	"math/big"
+	"strconv"
+
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	eth "github.com/smartcontractkit/chainlink-env/pkg/helm/ethereum"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver"
+	mockservercfg "github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver-cfg"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/contracts/ethereum"
+	"github.com/smartcontractkit/chainlink-testing-framework/utils"
+
+	networks "github.com/smartcontractkit/chainlink/integration-tests"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+)
+
+var _ = Describe("Automation OCR Suite @keeper", func() {
+	var (
+		err              error
+		chainClient      blockchain.EVMClient
+		contractDeployer contracts.ContractDeployer
+		registry         contracts.KeeperRegistry
+		registrar        contracts.KeeperRegistrar
+		consumers        []contracts.KeeperConsumer
+		upkeepIDs        []*big.Int
+		linkToken        contracts.LinkToken
+		chainlinkNodes   []*client.Chainlink
+		testEnvironment  *environment.Environment
+
+		testScenarios = []TableEntry{
+			Entry("v2.0 Basic smoke test @simulated", ethereum.RegistryVersion_2_0, defaultRegistryConfig, BasicCounter, BasicSmokeTest, big.NewInt(defaultLinkFunds), 2),
+		}
+	)
+
+	DescribeTable("Automation OCR Suite @keeper", func(
```

Comment:
same here

Reply:
same what here?

Reply:
i meant tag is @automation. (Same as comment on line 27)